xtask/codegen/src/generate_nodes.rs

```diff
@@ -976,7 +978,21 @@ pub(crate) fn token_kind_to_code(name: &str, language_kind: LanguageKind) -> Tok
     } else if kind_source.keywords.contains(&name) {
         // we need to replace "-" with "_" for the keywords
         // e.g. we have `color-profile` in css but it's an invalid ident in rust code
-        let token: TokenStream = name.replace('-', "_").parse().unwrap();
+        // also mark uppercase differently from lowercase
+        // e.g. "query" => "QUERY", "QUERY" => "Q_U_E_R_Y_"
```

Comment:
Why do we need to mark upper-case keywords differently? 
If so, is there a better way to tag them instead of using all underscores? Maybe with a postfix or something.

Reply:
Graphql have both uppercase (`QUERY`) and lowercase keyword (`query`) so I want to have a way to differentiate them. I don't know what is the best way to do this, maybe adding a postfix if all the letters are uppercase?

Reply:
> Graphql have both uppercase (`QUERY`) and lowercase keyword (`query`) so I want to have a way to differentiate them. I don't know what is the best way to do this, maybe adding a postfix if all the letters are uppercase?

I think this could help the readability.